crates/testing/src/test_runner.rs

```diff
@@ -311,7 +311,7 @@ where
                         }
                     }
                     Err(e) => {
-                        panic!("Error Joining the test task {:?}", e);
+                        tracing::error!("Error Joining the test task {:?}", e);
```

Comment:
Should this just be `error`

Reply:
Because this is just tokio if we add use tracing::error! it will cause lint errors for async_std.  I think it's easier to just have the fully qualified name here than to gate the imports